### ISSUE TITLE
Demo load-balancer breaks sessions by moving users between servers

### ISSUE THREAD

**Issue by dmos62** [mathesar-foundation/mathesar#2595]
@silentninja might provide more details (or correct me), but basically our demo load-balancer, when a server is determined to be overloaded, is moving users to new servers and that's breaking their sessions, because sessions are tied to servers.

**Comment by kgodey** [comment]
Do we need a separate issue for this? I think fixing this is covered under #2543 – we haven't actually moved demo.mathesar.org to the load balancer yet.

**Comment by dmos62** [comment]
@kgodey More comprehensive load testing (#2547) is blocked on this aspect of #2543. By tracking this I'll be notified when #2547 is unblocked. That's my reasoning for opening this. There are other ways to keep me up to date (and others observing these issues), but this one seems optimal to me.

**Comment by kgodey** [comment]
Okay sounds good, as long as @silentninja is also tracking this separately and will keep it up to date.

**Comment by kgodey** [comment]
This is related to our internal infrastructure, not Mathesar code. Moved to Basecamp [here](https://3.basecamp.com/5718119/buckets/35368592/todolists/6861165985).
